index.html

```diff
@@ -88,49 +96,224 @@
+  }
 
-  var svg;
+  function make_y_axis() {        
+    return d3.svg.axis()
+        .scale(y)
+        .orient("left")
+        .ticks(5)
+  }
 
-  // FETCH DATA
 
+  // FETCH DATA
+  d3.json('data.json', function (error, json) {
+    if (error) { throw error; }
+    data = json;
+    setup();
+    update();
+  });
   function setup() {
     // SVG
-
     svg = d3.select('#chart')
-      .append('svg');
-
+      .append('svg')
+      .attr('width', width + margin.right + margin.left)
+      .attr('height', height + margin.top + margin.bottom)
+      .append('g')
+      .attr('transform', 'translate(' + margin.left + ',' + margin.top + ')');
     // SCALES
-
+    x = d3.scale.linear()
+      .domain([0, d3.max(data, function (d) { return d.unemployment; })])
+      .range([0, width])
+      .nice();
+    y = d3.scale.linear()
+      .domain(d3.extent(data, function (d) { return d.employment; }))
+      .range([height, 0])
+      .nice();
+    
     // AXES
-
+    //Define X axis
+      xAxis = d3.svg.axis()
+                .scale(x)
+                .orient("bottom")
+                .ticks(5);
+    //Define Y axis
+      yAxis = d3.svg.axis()
+                .scale(y)
+                .orient("left")
+                .ticks(5);
     // EVENT HANDLERS
-
     d3.select('#year-range').on('change', function () {
       options.yearRange = d3.event.target.value;
       update();
     });
   }
-
   function update() {
     // TRANSFORM
     txData = data.slice();
-
     if (options.yearRange !== 'all') {
       var yearRange = options.yearRange.split('-');
       txData = txData.filter(function (d) {
         return d.year >= yearRange[0] && d.year <= yearRange[1];
       });
     }
-
     // SCATTERPLOT
+    var points = svg.selectAll('.point')
+      .data(data);
+    points.enter().append('circle')
+      .attr('class', 'point')
+      .attr('r', 5);
+    points
+      .attr('cx', function (d) { return x(d.unemployment); })
+      .attr('cy', function (d) { return y(d.employment); });
+    points.exit().remove();
+    
+    
+
+    // Creating (printing the Axxis)
+    //Create X axis
+      svg.append("g")
+        .attr("class", "axis")
+        .attr("transform", "translate(0," + (height - xpadding) + ")")
+        .call(xAxis)
+        .append("text")
+         .text("employment (%)")
+         .attr("x", width - 63)
+         .attr("y", 0)
+         .attr("font-family", "sans-serif")
+         .attr("font-size", "11px")
+         .attr("fill", "red");
+      
+      //Create Y axis
+      svg.append("g")
+        .attr("class", "axis")
+        .attr("transform", "translate(" + ypadding + ",0)")
+        .call(yAxis)
+        .append("text")
+          .text("unemployment (%)")
+          .attr("x", 0)
+          .attr("y", 0)
+          .attr("font-family", "sans-serif")
+          .attr("font-size", "11px")
+          .attr("fill", "blue");
+           
+      
 
+    //CALLING THE GRID
+    svg.append("g")         
```

Comment:
Note that you're adding two sets of axes. This is all possible with just a single x and a single y axis.

The `max_x_axis()` function is unnecessary. I'm not sure where you go that from, but it would perhaps be useful if you were to be making lots of axes.

What would happen if you were to delete lines 218–233, and just copy the `.tickSize()` and `.tickFormat()` calls onto the axes you define on lines 151–160?